hook/scriptwrapper.go

```diff
@@ -220,18 +220,29 @@ func (wrap *ScriptWrapper) Close() {
 
 // Changes returns the changes in the environment and working dir after the hook script runs
 func (wrap *ScriptWrapper) Changes() (HookScriptChanges, error) {
-	beforeEnvContents, err := ioutil.ReadFile(wrap.beforeEnvFile.Name())
+	beforeEnvContents, err := os.ReadFile(wrap.beforeEnvFile.Name())
```

Comment:
Ah I see [`ioutil.ReadFile` is deprecated](https://pkg.go.dev/io/ioutil@go1.19.2#ReadFile) 👍🏼

Reply:
all of ioutil is apparently! i learned this yesterday. we should probably go through and replace things, but the compatibility guarantee means that we probably won't have to until \*checks watch\* ever

Reply:
Was it golangci-lint that let you know?

Reply:
it might have been 😅